.travis.yml

```diff
@@ -1,14 +1,10 @@
 language: go
 sudo: required
-dist: trusty
-addons:
-  apt_packages:
-   - openjdk-7-jre-headless
+dist: xenial
```

Comment:
Upgrading to Ubuntu 16.04 seems reasonable since 14.04 will reach end of life next month.